src/PgRvn.Server/HardcodedQuery.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO.Pipelines;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using PgRvn.Server.Messages;
+using PgRvn.Server.Types;
+using TSQL;
+
+
+namespace PgRvn.Server
+{
+    public class HardcodedQuery : PgQuery
+    {
+        private PgTable _result;
+
+        public HardcodedQuery(string queryString, int[] parametersDataTypes) : base(queryString, parametersDataTypes)
+        {
+        }
+
+        public void Parse(bool allowMultipleStatements)
+        {
+            var sqlStatements = TSQLStatementReader.ParseStatements(QueryString);
```

Comment:
So we are using the SQL Parser? I thought we skipped it and used regexes?

Reply:
Only to recognize the number of statements in the SQL. Unfortunately `TSQLParser` doesn't even let us get the query string of each statement separately. 

If you want I can get rid of it and stop checking if there's more than 1 statement when not allowed (e.g. Extended Protocol only allows 1 query per transaction).

Reply:
I would rather avoid having a dependency on the SQL parser, yes.
And supporting just one is fine.